docs/two-phase-commit-transactions.md

```diff
@@ -5,6 +5,21 @@ With Two-phase Commit Transactions, you can execute a transaction that spans mul
 
 This document briefly explains how to execute Two-phase Commit Transactions in ScalarDB.
 
+## Overview
+
+ScalarDB transactions normally execute in a single transaction manager instance.
+In that case, you begin a transaction, execute CRUD operations, and commit the transaction in the same transaction manager instance.
+
+In addition to normal transactions, ScalarDB also supports two-phase-commit-style transactions called *Two-phase Commit Transactions*.
+Two-phase Commit Transactions execute a transaction that spans multiple transaction manager instances.
+The transaction manager instances can be in the same process/application or in different processes/applications.
+For example, if you have transaction manager instances in multiple microservices, you can execute a transaction that spans multiple microservices.
+
+In Two-phase Commit Transactions, there are two roles, a coordinator and a participant, that collaboratively execute a single transaction.
+A coordinator process and participant processes all have different transaction manager instances.
+The coordinator process first begins a transaction, and the participant processes join the transaction.
+After executing CRUD operations, the coordinator process and the participant processes execute the two-phase commit protocol to commit the transaction.
```

Comment:
```suggestion
After executing CRUD operations, the coordinator process and the participant processes commit the transaction by using the two-phase interface.
```
`the two-phase commit protocol` is not really defined here, so I felt it was difficult to parse.
I also thought it might be misleading since the protocols are both two-phase commits, but the interfaces are different in normal and two-phase commit transactions. so I suggested as above.